apps/resources/sbtn.json

```diff
@@ -6,5 +6,11 @@
     "org.scala-sbt:sbt:latest.stable"
   ],
   "launcherType": "graalvm-native-image",
-  "prebuilt": "zip+https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.zip!sbt/bin/sbtn-${platform}"
+  "prebuiltBinaries": {
+    "x86_64-pc-linux": "zip+https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.zip!sbt/bin/sbtn-x86_64-pc-linux",
+    "aarch64-pc-linux": "zip+https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.zip!sbt/bin/sbtn-aarch64-pc-linux",
+    "x86_64-apple-darwin": "zip+https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.zip!sbt/bin/sbtn-x86_64-apple-darwin",
+    "aarch64-apple-darwin": "zip+https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.zip!sbt/bin/sbtn-aarch64-apple-darwin",
```

Comment:
Note that that launcher (macOS on aarch64) is not yet produced by the sbt release workflow.